### ISSUE TITLE
Migrate example code and docs

### ISSUE THREAD

**Issue by bobturneruk** [FAIRDataPipeline/pyDataPipeline#56]
Migrate:

https://github.com/FAIRDataPipeline/pyDataPipeline/tree/dev/data_pipeline_api/ext

to

https://github.com/FAIRDataPipeline/pySimpleModel

Will impact CI on https://github.com/FAIRDataPipeline/FAIR-CLI

**Comment by bobturneruk** [comment]
This is where the CI needs to change:

https://github.com/FAIRDataPipeline/FAIR-CLI/blob/d7c28ffa2b152bb4bc99ab27513cf95c39bb2d4d/.github/workflows/implementations.yml#L12

**Comment by bobturneruk** [comment]
Cheers @bruvio - when I get to the point of making a commit, I'll be ready for a draft PR! 😄

**Comment by bobturneruk** [comment]
Currently trying to run pySimpleModel following these instructions (which I just wrote):

## Installation

1. Create a and activate new python virtual environment.
2. `pip install fair-cli`.
3. Clone this repository: `git clone https://github.com/FAIRDataPipeline/pySimpleModel.git`.
4. Change directory `cd pySimpleModel`.
5. Install this package `pip install -e .`.

**Comment by bobturneruk** [comment]
Getting:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
data-pipeline-api 0.7.5 requires PyYAML<7.0,>=6.0, but you have pyyaml 5.4.1 which is incompatible.
```

**Comment by bobturneruk** [comment]
Or I can chose:
```
pip install pyyaml==6.0
```

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
fair-cli 0.2.3 requires PyYAML<6.0.0,>=5.4.1, but you have pyyaml 6.0 which is incompatible.
```

**Comment by bruvio** [comment]
this error is ok as far as I know and the model is installed.

**Comment by bruvio** [comment]
https://github.com/FAIRDataPipeline/pyDataPipeline/runs/5677886707?check_suite_focus=true

**Comment by bruvio** [comment]
that is a successful pipeline where the error you bumped in occurs, so no worries

**Comment by bobturneruk** [comment]
Is it not a clash between the api and the cli?

**Comment by RyanJField** [comment]
> Is it not a clash between the api and the cli?

Yes, it's the API, once merged with develop this should be solved. the API has dependabot updating the dependencies, I must have merged a new version of pyyaml without realising it was too far above the CLI.
